information-schema/information-schema-slow-query.md

```diff
@@ -7,6 +7,14 @@ summary: Learn the `SLOW_QUERY` information_schema table.
 
```

Comment:
Can you add these limitations in https://docs.pingcap.com/tidbcloud/serverless-tier-limitations